### ISSUE TITLE
AdF.ly is not loading, preventing me from creating a room

### ISSUE THREAD

**Issue by salmonofdoubt12** [aeosynth/draft#133]
This might be an issue that needs to be fixed locally on my computer, but I talked to a few other people with the same problem and I can't seem to figure it out. I'm using Chrome.

Basically, after creating a draft or sealed on Drafts.in I get sent to the Ad.Fly page. Instead of giving me the usual 5 second countdown and allowing me to skip the ad, I get a pop-up box saying "Confirm Navigation" and letting me either "Stay on this Page" or "Leave this Page." If I chose to stay on the page, the page says that my cookies are not enabled and that I need Flash installed. The "please wait" in the top right corner never starts a countdown and I can't create a room. If I choose to leave the page, I get redirected to another ad that doesn't load without the "please wait" in the top right corner.

When I went into my Chrome settings, cookies were enabled. Just to make sure, I reenabled them and made an exception for both Drafts.in and Ad.Fly. I even turned off adblock for both sites too. I also made sure that my Flash player was updated (it was). I have no idea how to fix it, and I can no longer create rooms on Drafts.in.

**Comment by aeosynth** [comment]
just tried and it works for me on chrome, is this still an issue for you? try completely disabling your adblock. i could switch to a different ad network.

**Comment by salmonofdoubt12** [comment]
I attached some screenshots showing what I'm talking about.  I cleared my history, cookies, and made exceptions for the relevant sites. I turned off adblock. 
![screen shot 2015-09-20 at 6 39 59 pm](https://cloud.githubusercontent.com/assets/8680625/9983447/1b82cfcc-5fca-11e5-9094-65042a451592.png)
![screen shot 2015-09-20 at 6 41 46 pm](https://cloud.githubusercontent.com/assets/8680625/9983449/1f10f65a-5fca-11e5-8763-e347c5ede139.png)

No idea why it's still not working.
